src/types/entities.ts

```diff
@@ -0,0 +1,112 @@
+import { EntityFromIntegration } from '@jupiterone/jupiter-managed-integration-sdk';
+
+export interface StandardizedOktaAccount extends EntityFromIntegration {
```

Comment:
I'd like to see all these Entity sub-types go away in the new integration. I'm convinced it's an anti-pattern to have to create/maintain these, since it implies that folks will be referencing these properties statically in code. We can talk more about this to be sure, but take a look at https://github.com/JupiterOne/graph-azure/pull/258#discussion_r602459779.

Reply:
Removed